### ISSUE TITLE
Add shortcut to open code faster in an editor

### ISSUE THREAD

**Issue by diegozuber** [ForkIssues/Tracker#1107]
I would like to see a shortcut with which I can open code with a keyboard shortcut in VSCode (or any other Editor).

**Comment by DanPristupov** [comment]
<img width="398" alt="Screen Shot 2020-08-17 at 11 58 56" src="https://user-images.githubusercontent.com/618115/90383873-08b98380-e081-11ea-8545-8aaf329716f7.png">

You can press ⌥⌘⇧O to open file in the default editor.

**Comment by diegozuber** [comment]
@DanPristupov Yes, but I'm talking about a whole repo. Like you can do it now in the UI:

![Bildschirmfoto 2020-08-17 um 12 03 11 PM](https://user-images.githubusercontent.com/21689209/90384371-ba58b480-e081-11ea-8f48-217791364965.png)

**Comment by DanPristupov** [comment]
VSCode is not the only application in this list. It can be VSCode, Atom, Sublime, XCode, etc at the same time.

**Comment by diegozuber** [comment]
@DanPristupov Maybe like this:

 ⇧⌘1 VSCode
 ⇧⌘2 Atom
 ⇧⌘3 Sublime
 ⇧⌘4 XCode
...and so on?

**Comment by igreg** [comment]
I think it would be reasonable to allow users to specify a default editor and for you to provide a shortcut to open the active repository with it. Yes you can have multiple editors, but realistically most people will be happy to choose one to use by default.

I really like Fork and this is maybe the only thing that I miss from Github Desktop.

**Comment by igreg** [comment]
Alternatively, if you could add "Open In" with its options under "Repository" in the menubar, then we could use MacOS to add shortcuts ourselves as a work around.

**Comment by ian-h-chamberlain** [comment]
> Alternatively, if you could add "Open In" with its options under "Repository" in the menubar, then we could use MacOS to add shortcuts ourselves as a work around.

@igreg – I believe you should still be able to use those options even without it appearing in the top menu bar... this configuration seems to work for me:

<img width="448" alt="Screenshot 2024-06-12 at 12 31 30" src="https://github.com/fork-dev/Tracker/assets/11131775/f12beca1-0b26-40ae-9893-0e54b38dee4c">

The shortcut even appears in the `Open In` menu:
<img width="328" alt="Screenshot 2024-06-12 at 12 32 54" src="https://github.com/fork-dev/Tracker/assets/11131775/6be6d025-9eeb-4ae1-81f2-1c517ee30aca">

I assume this would extend to other applications people might want to open, hopefully that satisfies this request!
